client/src/store/store.js

```diff
@@ -0,0 +1,20 @@
+import { createStore } from 'redux';
+import rootReducer from './reducers/reducers';
+
+const localStorageKey = 'theme';
+const persistedTheme = localStorage.getItem(localStorageKey);
+
+let initialState = {
+  darkModeReducer: persistedTheme ? JSON.parse(persistedTheme) : {}
```

Comment:
it's not a reducer, it's state, so naming is a bit confusing. please make it simpler. 
also please make sure your initial state has the same structure as state you use in reducers